### PR TITLE
An improvement to the "Incorrect usernames warning feature" dialog

### PR DESCRIPTION
**Pull request by NdibeRaymond**
Issue fixed: #197 

Sample image:
modal_warning_dialog

### CODE REVIEW

File: frontend/WikiContrib-Frontend/src/query.js [hunk 1]
```diff
@@ -414,6 +465,42 @@ export class Query extends Component {
...
+              >
+              <Modal.Header>Warning!
+                <Icon
+                name="times"
+                className="times-icon"
+                onClick={()=> this.setState({showMismatch:true,loading:false})}
+              />
+              </Modal.Header>
+              <Modal.Content>
+                <h4>The usernames you have provided does not seem to belong to the same user.
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:482]: Enclose this in a <p> tag instead to reduce the boldness.

File: frontend/WikiContrib-Frontend/src/query.js [hunk 2]
```diff
@@ -414,6 +465,42 @@ export class Query extends Component {
...
           <React.Fragment>
             <NavBar />
+            <Modal
+              id="fuzzy-warning"
+              open={!this.state.showMismatch}
+              onClose={()=> this.setState({showMismatch:true,loading:false})}
+              size='tiny'
+              >
+              <Modal.Header>Warning!
+                <Icon
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:475]: This icon, I think we can either remove or modify such that hovering on it changes the pointer to a hand cursor. I'm inclined for just removing it as we have a "No" button that does the same. Whichever is easy implementation wise.

File: frontend/WikiContrib-Frontend/src/query.js [hunk 3]
```diff
@@ -326,42 +337,82 @@ export class Query extends Component {
...
+        loadData: false,
+        message: {
+          message: response.message,
+          trigger: true,
+          type: 1,
+          update: !this.state.message.update,
+        },
+      });
+    }
+    else{
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:406]: Space between else and { and similar changes to the else block below

File: frontend/WikiContrib-Frontend/src/query.js [hunk 4]
```diff
@@ -326,42 +337,82 @@ export class Query extends Component {
...
+              this.callback
+            );
+          }
+      }
       }
     }
   };
 
+  matchFullNamesCallback = response => {
+    if(response !== '' &&
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:392]: Space between if and (

File: frontend/WikiContrib-Frontend/src/query.js [hunk 5]
```diff
@@ -326,42 +337,82 @@ export class Query extends Component {
...
+    response !== 'error' in response &&
+    response.error === 1){
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:394]: Add a space between ) and {

File: frontend/WikiContrib-Frontend/src/query.js [hunk 6]
```diff
@@ -326,42 +337,82 @@ export class Query extends Component {
...
-        ) {
-          this.callback('');
-        } else {
-          let data = { file: -1 };
-          data['users'] = this.getExactRows();
-          this.setState({ loadData: true, notfound: false });
-          fetchAsynchronous(
-            uri,
+
+      }else{
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:353]: Add a space between } and else and else and {

File: frontend/WikiContrib-Frontend/src/query.js [hunk 7]
```diff
@@ -326,42 +337,82 @@ export class Query extends Component {
...
       this.setState({ loading: true, notfound: false });
-      let uri = this.state.operation
-        ? QueryCreateApi
-        : QueryDetailApi.replace('<hash>', this.props.match.params.hash);
-      if (this.state.file !== false) {
-        fetchFileAsynchronous(
-          this.state.file,
-          this.state.operation ? 'POST' : 'PATCH',
+
+      if(match_fullnames){
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/query.js:341]: Add space between if and ( and ) and {

File: backend/WikiContrib/result/views.py [hunk 8]
```diff
@@ -133,6 +135,66 @@ async def get_full_name(data):
...
+        if isinstance(gerrit_username, float):
+            gerrit_username = ''
+
+        if isinstance(github_username, float):
+            github_username = ''
+
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+
+        full_names = {"phab_full_name":"","gerrit_full_name":"","github_full_name":""}
```
Comment (srish) [inline backend/WikiContrib/result/views.py:174]: add a space after every ,

File: backend/WikiContrib/result/views.py [hunk 9]
```diff
@@ -133,6 +135,66 @@ async def get_full_name(data):
...
+        request_data.append(deepcopy(REQUEST_DATA[3]))
+
+        urls.append(API_ENDPOINTS[0][1])
+        urls.append(API_ENDPOINTS[1][1].format(gerrit_username=gerrit_username))
+
+        request_data[0]['constraints[usernames][0]'] = phabricator_username
+
+        request_data[1]['github_username'] = github_username
+
+        loop.run_until_complete(concurrent_get_fullnames(urls=urls, request_data=request_data,
```
Comment (srish) [inline backend/WikiContrib/result/views.py:188]: Same here, consistent spacing after ,

File: frontend/WikiContrib-Frontend/src/query.js (hunk 1 above)
Comment (NdibeRaymond) [reply frontend/WikiContrib-Frontend/src/query.js:482]: Ok

File: frontend/WikiContrib-Frontend/src/query.js (hunk 2 above)
Comment (NdibeRaymond) [reply frontend/WikiContrib-Frontend/src/query.js:475]: I think removing is better. It is always better to reduce code when possible than add to it

**Pull request merged**